app/src/__stories__/StoryWrapper.tsx

```diff
@@ -70,13 +70,13 @@ const StoryWrapper: React.FC<{
   return (
     <StoreProvider store={store}>
       <ThemeProvider>
-        <Router history={store.router.history}>
+        <BrowserRouter>
```

Comment:
Using `BrowserRouter` instead of generic `Router` as suggested in their [guidance](https://reactrouter.com/docs/en/v6/routers/router).